hatch.toml

```diff
@@ -64,7 +64,7 @@ dependencies = [
   # Plugins
   "mkdocs-minify-plugin~=0.5.0",
   "mkdocs-git-revision-date-localized-plugin~=1.1.0",
-  "mkdocstrings-python~=0.7.1",
+  "mkdocstrings[python]>=0.18",
```

Comment:
Let's keep:

1. I'm strict with docs deps to avoid breakages
2. Docs require [0.7.0](https://mkdocstrings.github.io/python/changelog/#070-2022-05-28) which [this](https://github.com/mkdocstrings/mkdocstrings/blob/0.19.0/pyproject.toml#L44) does not guarantee